Import/Writer/Writer.php

```diff
@@ -57,7 +57,7 @@ public function __construct(RedirectRouteManagerInterface $manager, EntityManage
     public function write(RedirectRouteInterface $entity)
     {
         $this->validate($entity);
-        $this->sources[] = $entity->getSource();
+        $this->sources[] = strtolower($entity->getSource());
```

Comment:
/cc @chirimoya

Reply:
the problem here is that the unique key of mysql is case-insensitive by default. we could add this here or mark the unique key as `BINARY`